src/tool/abi/main.cpp

```diff
@@ -21,9 +21,57 @@ auto output_directory(reader const& args)
     return out.string();
 }
 
-void print_usage()
+struct usage_exception {};
+
+static constexpr option options[]
+{
+    { "input", 0, option::no_max, "<spec>", "Windows metadata to include in generation" },
+    { "reference", 0, option::no_max, "<spec>", "Windows metadata to reference from generation" },
+    { "output", 0, 1, "<path>", "Location of generated headers" },
+    { "include", 0, option::no_max, "<prefix>", "One or more prefixes to include in input" },
+    { "exclude", 0, option::no_max, "<prefix>", "One or more prefixes to exclude from input" },
+    { "verbose", 0, 0, {}, "Show detailed progress information" },
+    { "ns-prefix", 0, 1, "<always|optional|never>", "Set namespace prefixing preference (default: never)" },
```

Comment:
>Set namespace prefixing preference (default: never) [](start = 53, length = 51)

Should probably call out that the prefix is the `ABI::` namespace prefix. E.g. maybe
> Controls whether or not type names are prepended with the 'ABI' namespace

For comparison, here's all that midl says: "`Prepend the 'ABI' namespace to all types`"